app/models/request.rb

```diff
@@ -0,0 +1,4 @@
+class Request < ApplicationRecord
+  belongs_to :requestedBy, :class_name => 'User'
```

Comment:
can we use the convention here: `requested_by` & `owned_by`. Rails may have issues later if those are not set that way

Reply:
or should we use `borrower` & `owner` instead if that reads better???